lib/screens/follow_up_screen/edit_follow_up_screen.dart

```diff
@@ -208,6 +208,9 @@ class _EditFollowUpScreenState extends State<EditFollowUpScreen> {
             decoration: InputDecoration(
               labelText: 'Syst (50-250 mmHg)',
               border: const OutlineInputBorder(),
+              focusedBorder: OutlineInputBorder(
```

Comment:
Maybe you can create const variable "focusedBorder" befor return Row.. and use it everywhere in this class.